app/views/idv/doc_auth/_start_over_or_cancel.html.erb

```diff
@@ -1,9 +1,8 @@
-<br/>
```

Comment:
These changes will affect the live screen, yeah? Doesn't seem like it should be an issue, but flagging just-in-case.

Reply:
It was affecting my ability to match the design. This will affect the current screens, but not to the extent that they break anything:

![Screen Shot 2020-07-30 at 11 39 04 AM](https://user-images.githubusercontent.com/2583060/88943317-52206b00-d259-11ea-876e-17b42f080fa5.png)